build.sbt

```diff
@@ -89,13 +89,13 @@ val s3Upload = TaskKey[PutObjectResult]("s3-upload", "Uploads files to an S3 buc
 lazy val root: Project = (project in file("."))
   .enablePlugins(GitVersioning)
   .settings(buildSettings)
-  .settings(name := "elasticmq-root", publishArtifact := false)
-  .aggregate(commonTest, core, rest, persistence, server, nativeServer, ui)
+  .settings(name := "elasticmq-root", publish / skip := true)
+  .aggregate(commonTest, core, rest, persistence, server, ui)
```

Comment:
shouldn't `nativeServer` remain in aggregates?

Reply:
Here's the tricky part, we want to build the main jar using java 8, but native-server requires java 11, so it's built separately. Besides, there's nothing to build/test there within the main pipeline so it can be safely removed from root. It's still used when building native image ofc.

Reply:
aah ok - let's add a comment why it's not in the aggregates, it's non-obvious :)